Gems/Atom/Utils/Code/atom_utils_editor_files.cmake

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    Include/Atom/Utils/AssetSystemStub.h
```

Comment:
Should this be in a test folder to make things more clear?
Include/Atom/Utils/TestUtils/AssetSystemStub.h